spec/system/authentication_and_authorisation_spec.rb

```diff
@@ -1,6 +1,37 @@
 require "spec_helper"
 
 RSpec.describe "Authenication and authorisation" do
+  context "omniauth request phase" do
+    let(:authorize_params) { Rack::Utils.parse_query(URI.parse(page.response_headers["Location"]).query) }
```

Comment:
Minor: I think it might be slightly clearer to extract `URI.parse(page.response_headers["Location"])` as a separate `let` (e.g. `redirect_uri`) and add an extra test/assertion that the path is "/oauth/authorize" as mentioned in the names of the two tests you've added.

Reply:
Done :+1: